### ISSUE TITLE
Restore doesn't work

### ISSUE THREAD

**Issue by havardgulldahl** [havardgulldahl/jottalib#74]
Seems like Jottacloud has gone and changed the web api, and `restore` and maybe other methods relying on the web api have stopped working.

Tagging @thusoy, since you originally did the hard work on this.

**Comment by thusoy** [comment]
Yes, I have noticed. Not sure how much has changed, I haven't had time to study the new api yet.

**Comment by thusoy** [comment]
Hm, the issues I had noticed had actually already been fixed in jottalib (related to incomplete files), so I haven't seen any issues with the restore. Care to elaborate what sort of errors you're seeing, @havardgulldahl?

**Comment by havardgulldahl** [comment]
Unfortunately, it seems they have shut down the old api. The web client is obfuscating the paths, making this a pain to figure out. We should figure out how the client does this now. 

@jkaberg, are you up to providing some more logs? *delete+restore* of files and folders, and maybe also *hard_delete*, if that is possible in the client..

**Comment by havardgulldahl** [comment]
@jkaberg ping. If you help me out here, I'll take a look at Fuse+EncFS ;)

**Comment by jkaberg** [comment]
@havardgulldahl Sure I'll fetch some logs for you, only restore? Missing anything else?

**Comment by havardgulldahl** [comment]
Great! 
I think you maybe have to use the mobile client to find *delete&restore* functionality to capture. I can't see it in the desktop client. 

Could you also do 

*search+get url*, a.k.a. "share".

That, at least, is possible in the desktop client. Thanks a bunch!

**Comment by jkaberg** [comment]
The mobile client involves MITM attack, might not be so easy to pull of (requires root with android aswell). I'll have to think about this one, on how best to proceed. 

The desktop client isn't an problem

Mandag, September 05, 2016 22:18 CEST skrev Håvard Gulldahl <notifications@github.com>:
 
Great!
I think you maybe have to use the mobile client to find delete&restore functionality to capture. I can't see it in the desktop client.
Could you also do
search+get url, a.k.a. "share".
That, at least, is possible in the desktop client. Thanks a bunch!

**Comment by thusoy** [comment]
MitMing the mobile client shouldn't be a problem unless they pin their certificates. Have you checked whether they do that?

**Comment by jkaberg** [comment]
So just looked into the desktop part, seem's the context menu options are removed outside of the "Jotta" folder. When hovering the Jottacloud option outside the jotta folder you get nothing, it's just a stub.
